### PR TITLE
Allow Pad Factory to put advancement tokens on Runner cards

### PR DESCRIPTION
**Pull request by jwarwick**
Fix for Issue #2728

### CODE REVIEW

File: src/clj/game/cards/assets.clj [hunk 1]
```diff
@@ -988,8 +988,7 @@
    "PAD Factory"
    {:abilities [{:cost [:click 1]
                  :label "Place 1 advancement token on a card"
-                 :choices {:req #(and (= (:side %) "Corp")
-                                      (installed? %))}
+                 :choices {:req #(installed? %)}
```
Comment (JoelCFC25) [inline src/clj/game/cards/assets.clj]: This works fine--it can also be as simple as `{:req installed?}`

**Comment by JoelCFC25** [comment]
Another idea to work in conjunction with this--Red Planet Couriers needs to scan all Runner cards for advancements in addition to Corp cards!

File: src/clj/game/cards/assets.clj (hunk 1 above)
Comment (jwarwick) [reply src/clj/game/cards/assets.clj]: Cool, I assumed it has to be the whole anon function. I've simplified it in the latest commit.

**Comment by jwarwick** [comment]
Looking at the code, it seems like there are a few cards that might need to be addressed: Shipment from Tennin, Dedication Ceremony.

Mass Commercialization appears to consider both sides already.

**Comment by JoelCFC25** [comment]
Tennin can target Runner cards already. Dedication Ceremony would need the tweak:  Runner and installed OR Corp and rezzed

**Comment by jwarwick** [comment]
I updated Dedication Ceremony and Red Planet Couriers.

I'm new to Netrunner, so I don't understand if I'm reading Shipment from Tennin incorrectly or reading the code wrong. It doesn't look like I can place advancement tokens on Runner cards currently.

**Comment by mtgred** [comment]
👍

**Pull request merged**